### PR TITLE
Updates for alignment with Jakarta 11 

### PR DESCRIPTION
**Pull request by spericas**
Signed-off-by: Santiago Pericasgeertsen <santiago.pericasgeertsen@oracle.com>

## Specification PR template
When creating a specification project release review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] Update of _index.html to reflect alignment with Jakarta 11

### CODE REVIEW

File: restful-ws/4.0/_index.md [hunk 1]
```diff
@@ -1,32 +1,38 @@
...
-    * Support for multipart media type - [Issue #418](https://github.com/eclipse-ee4j/jaxrs-api/issues/418)
-    * Better alignment with JSON-B - [Issue #742](https://github.com/eclipse-ee4j/jaxrs-api/issues/742)
-    * Automatic loading of provider extensions - [Issue #751](https://github.com/eclipse-ee4j/jaxrs-api/issues/751)
-    
-The JDK version required will be aligned with Jakarta EE 10. The latest details and progress for this work 
+* Explore alignment with CDI lite and potentially built-time injection support - [Issue #1137](https://github.com/eclipse-ee4j/jaxrs-api/issues/1137)
+
+### Removals, deprecations or backwards incompatible changes
+* Drop support for @Context injection and related artifacts - [Issue #951](https://github.com/eclipse-ee4j/jaxrs-api/issues/951)
+* Review integration with legacy Jakarta EE specifications (e.g., EJB) - [Issue #954](https://github.com/eclipse-ee4j/jaxrs-api/issues/954)
```
Comment (Pandrex247) [inline restful-ws/4.0/_index.md:23]:
```suggestion
* Review integration with legacy Jakarta EE specifications (e.g. EJB) - [Issue #954](https://github.com/eclipse-ee4j/jaxrs-api/issues/954)
```

**Comment by arjantijms** [comment]
Links to issues should be adjusted to `https://github.com/jakartaee/rest`. They currently link to the old location.

File: restful-ws/4.0/_index.md [hunk 2]
```diff
@@ -1,32 +1,38 @@
...
 Jakarta RESTful Web Services provides a foundational API to develop web services following the Representational
 State Transfer (REST) architectural pattern.
 
 The goal of this release is to provide better alignment between Jakarta RESTful Web Services and 
 Jakarta Contexts Dependency Injection (CDI). This will be the first _backward incompatible_ release of 
 Jakarta REST.  
 
-* Drop support for @Context injection and related artifacts - [Issue #951](https://github.com/eclipse-ee4j/jaxrs-api/issues/951)
+### New features, enhancements or additions
 * Define CDI scopes for all Jakarta REST types - [Issue #952](https://github.com/eclipse-ee4j/jaxrs-api/issues/952)
```
Comment (arjantijms) [inline restful-ws/4.0/_index.md:15]: https://github.com/eclipse-ee4j/jaxrs-api/issues/952 should be https://github.com/jakartaee/rest/issues/952

**Comment by spericas** [comment]
@ivargrimstad Thanks for addressing the comments from the reviewers

**Pull request merged**